src/Microsoft.Health.Fhir.Api/Features/ApiNotifications/ApiNotificationMiddleware.cs

```diff
@@ -64,7 +64,7 @@ protected virtual async Task PublishNotificationAsync(HttpContext context, Reque
                 }
                 finally
                 {
-                    apiNotification.Latency = timer.GetElapsedTime();
+                    apiNotification.Latency = timer?.ElapsedTime ?? TimeSpan.Zero;
```

Comment:
>timer [](start = 46, length = 5)

[https://github.com/microsoft/healthcare-shared-components/blob/master/src/Microsoft.Health.Core/ActionTimerExtensions.cs](https://github.com/microsoft/healthcare-shared-components/blob/master/src/Microsoft.Health.Core/ActionTimerExtensions.cs)
Am I missing something, or it's for defensive purpose?

Reply:
True, I wouldn't think `timer` would be null within the using scope

Reply:
With the package updates needed, it looks like `GetElapsedTime()` is now a property. The null check was just to get rid of the warning...

Reply:
Removed the null check in 29af3c4.